istio/README.md

```diff
@@ -141,6 +141,45 @@ You can use the Openmetrics V2 implementation of the Istio integration to resolv
 Note: you must upgrade to at minimum Agent `7.31.0` and Python 3. See the [Configuration](#configuration) section on enabling Openmetrics V2.
 
 
+### Using the generic Openmetrics Integration in an Istio deployment
+
+If Istio proxy sidecar injection is enabled, monitoring other Prometheus metrics via the [Openmetrics integration][21] with the same metrics endpoint as `istio_mesh_endpoint` can result in high custom metrics usage and duplicated metric collection.
+
+To ensure that your Openmetrics configuration does not redundantly collect metrics, either:
+
+1. Use specific metric matching in the `metrics` configuration option, or
+2. If using the wildcard `*` value for `metrics`, consider using the following Openmetrics integration options to exclude metrics already supported by the Istio and Envoy integrations.
+
+#### Openmetrics V2 configuration with generic metric collection
+
+Be sure to exclude Istio and Envoy metrics from your configuration to avoid high custom metrics billing. Use `exclude_metrics` if configuring the Openmetrics V2 (`openmetrics_endpoint` enabled).
```

Comment:
```suggestion
Be sure to exclude Istio and Envoy metrics from your configuration to avoid high custom metrics billing. Use `exclude_metrics` if using the Openmetrics V2 configuration (`openmetrics_endpoint` enabled).
```